src/utils/logger.ts

```diff
@@ -0,0 +1,11 @@
+import Pino from "pino"
+import config from "../lib/config"
+
+const logger: Pino.Logger = Pino({
+  name: `user-service-${process.env.NODE_ENV}`,
+  level: config.debugMode === "true" ? "debug" : "info",
+  timestamp: false,
```

Comment:
I figured we've got timestamps applied on the logs in kibana already so we don't really need them here too

Reply:
How long does it take to get logs from the user-service container into Kibana?

For example, I know that cloudwatch is "near real-time", and there can often be a delay of several minutes before logs appear. It would be good if this delay didn't affect the timestamps of the log messages?